binaryutil/binaryutil.go

```diff
@@ -102,3 +103,22 @@ func (bigEndian) Uint32(b []byte) uint32 {
 func (bigEndian) Uint64(b []byte) uint64 {
 	return binary.BigEndian.Uint64(b)
 }
+
+// For dealing with types not supported by the encoding/binary interface
```

Comment:
add a blank line after this comment to avoid it becoming the documentation of PutInt32.